### PR TITLE
Gather several commands in one binary

### PR DESCRIPTION
**Pull request by gabrielhdt**
Replaced commands `dkcheck`, `dkdep`, `dkmeta`, `dkprune` and `dktop` by `dk check`, `dk dep`, `dk meta`, `dk prune` and `dk top`.

### CODE REVIEW

**Comment by GuillaumeGen** [comment]
I think that if we want to have a binary for all the commands, this binary should be name `dedukti` not `dkcheck`.

**Comment by gabrielhdt** [comment]
> I think that if we want to have a binary for all the commands, this binary should be name `dedukti` not `dkcheck`.

I agree, but Dedukti should become a standardised _language_, while this ocaml piece of code should only be one implementation of it. I agree that `dedukti check` would make more sense than `dkcheck check`, but it has been decided (in a team meeting) that the implementation should be called `dkcheck`. Personally I have no strong opinion on that.

**Comment by GuillaumeGen** [comment]
Ok, if it is the result of a consensus and not simply something you overlooked, then forget my comment.

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ default: bin binaries
 all: bin binaries universo
 
 .PHONY: binaries
-binaries: dkcheck.native dktop.native dkdep.native dkprune.native dkmeta.native dkpretty.native
+binaries: dkcheck.native dktop.native dkprune.native dkmeta.native
```
Comment (fblanqui) [inline Makefile:14]: Do you plan to remove dkdep?? I use it for testing lp2dk.

Comment (gabrielhdt) [reply Makefile:14]: I guess, since it is replaced with `dkcheck dep`. An update will be required on lp2dk's side.

**Comment by francoisthire** [comment]
Instead of `dkcheck` we could use `dk`

**Comment by francoisthire** [comment]
This MR should also modify accordingly the `dedukti.ml` file for `Tezt`.

**Comment by gabrielhdt** [comment]
Should be good now.

**Review by francoisthire — APPROVED** [review]

**Pull request merged**